Algebraic_kernel_d/include/CGAL/RS/dyadic.h

```diff
@@ -13,8 +13,12 @@
 
 #include <stdio.h>
 #include <math.h>
+#ifdef CGAL_USE_GMP
 #include <gmp.h>
+#endif
+#ifdef CGAL_USE_MPFR
```

Comment:
This does not make much sense to me, the file uses MPFR unconditionally, removing the includes won't help if you try to include this file without MPFR.